src/vs/vscode.d.ts

```diff
@@ -7375,6 +7375,8 @@ declare module 'vscode' {
 	 */
 	export namespace languages {
 
+		export function changeLanguage(document: TextDocument, languageId: string): Thenable<void>;
```

Comment:
We should start with proposed API. Please move this to `vscode.proposed.d.ts` (inside a languages-namespace).